miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/MiniAppStatus.kt

```diff
@@ -5,11 +5,17 @@ import android.content.SharedPreferences
 import androidx.annotation.VisibleForTesting
 import java.io.File
 
+private const val LOCAL_VERSION = "local_version"
+
 internal class MiniAppStatus(context: Context) {
     private val prefs: SharedPreferences = context.getSharedPreferences(
         "com.rakuten.tech.mobile.miniapp.storage", Context.MODE_PRIVATE
     )
 
+    var localVersion: String?
+        get() = prefs.getString(LOCAL_VERSION, null)
+        set(downloadedVersion) = prefs.edit().putString(LOCAL_VERSION, downloadedVersion).apply()
```

Comment:
I think I'm not understanding something here. This is aways saving the local version to the same preferences key, so it will overwrite the cached version regardless of the mini app id, right? So how would this work when multiple mini apps have been downloaded?

Reply:
I'm sorry 🙏 
I have updated code for caching multiple apps.

Reply:
Ah, okay no problem! I thought I was missing seeing something 😆